photon-client/src/main/scala/com/linkedin/photon/ml/hyperparameter/VectorRescaling.scala

```diff
@@ -26,54 +26,6 @@ object VectorRescaling {
   val LOG_TRANSFORM = "LOG"
   val SQRT_TRANSFORM = "SQRT"
 
-  /**
-   * Apply forward transformation to a subset of elements in a vector.
-   *
-   * @param vector A DenseVector.
-   * @param transformMap A Map with key-value pairs of indices and names of transform functions.
-   * @return The transformed vector.
-   */
-  def transformForward(
```

Comment:
Why do we delete this function? Our internal atlas-ml does not have functions to perform parameter transformation.

Reply:
Let's wait to merge this PR until the internal dependency changes.

Reply:
@yunboouyang @cmjiang I don't understand this comment - is there no equivalent in the internal library for these functions? If so, is there a blocker to copying the implementation from here?

Reply:
@yunboouyang @cmjiang 
I don't understand this comment - are these two functions not available in the internal hyperparameter tuning library? Is there a blocker to copying the implementation over from here?

Reply:
The reason to leave these two functions in Photon ML is that, we wanted to do the same transformation in Photon ML. Photon ML doesn't have the dependency on the internal hyperparameter tuning library. If Photon ML doesn't need to reuse these functions, we can move it to the tuning library.